src/main/java/com/alipay/oceanbase/hbase/OHTable.java

```diff
@@ -1127,6 +1147,21 @@ public void setOperationTimeout(int operationTimeout) {
             (this.operationTimeout != HConstants.DEFAULT_HBASE_CLIENT_OPERATION_TIMEOUT));
     }
 
+    @Override
+    public int getOperationTimeout() {
+        return 0;
+    }
```

Comment:
add todo comment or throw not support exception

Reply:
fix